package.json

```diff
@@ -91,6 +91,7 @@
     "build": "NODE_OPTIONS=--max-old-space-size=4096 IMAGE_INLINE_SIZE_LIMIT=0 INLINE_RUNTIME_CHUNK=false DISABLE_ESLINT_PLUGIN=true react-app-rewired build && node headers.js && ./scripts/sha256sums.sh && ./scripts/verifyWebpackHashes.sh",
     "dev": "NODE_OPTIONS=--max-old-space-size=4096 ESLINT_NO_DEV_ERRORS=true TSC_COMPILE_ON_ERROR=true IMAGE_INLINE_SIZE_LIMIT=0 INLINE_RUNTIME_CHUNK=false react-app-rewired start",
     "dev:silent": "NODE_OPTIONS=--max-old-space-size=4096 ESLINT_NO_DEV_ERRORS=true TSC_COMPILE_ON_ERROR=true IMAGE_INLINE_SIZE_LIMIT=0 INLINE_RUNTIME_CHUNK=false BROWSER=none react-app-rewired start",
+    "local-ci": "yarn lint && yarn type-check && yarn test && yarn build",
```

Comment:
```suggestion
    "ci": "yarn lint && yarn type-check && yarn test && yarn build",
```

Reply:
after discussing this, did you want me to rename this to "fake-local-ci" or similar, or is this ok to merge? i was trying to avoid any ambiguity that this is actually what CI does as it's defined by the `pr.yml` file invoking things in steps